index.html

```diff
@@ -1777,7 +1777,10 @@ <h2 id="Collection">Backbone.Collection</h2>
     <p id="Collection-remove">
       <b class="header">remove</b><code>collection.remove(models, [options])</code>
       <br />
-      Remove a model (or an array of models) from the collection, and returns them.
+      Remove a model (or an array of models) from the collection, and returns
+      them. <tt>models</tt> (or each of its elements if an array) can be any
```

Comment:
I think it's preferable to keep this DRY and refer to the `id` arg of `get()` for the acceptable values instead of duplicating them here. Your call.

I think it's better to be explicit that `models` can be a a single item or an array, so I feel that replacing "a model (or an array of models)" with "The model or models" reduces clarity.

How about?:
```
Remove model(s) and return a model, or array of models (if `models` is an
array). `models` can be any value acceptable as the <tt>id</tt> argument of
<a href="#Collection-get"><tt>collection.get</tt></a>, or an array of those values.
```